### PR TITLE
Implement the Climber

### PR DESCRIPTION
**Pull request by terbos90803**
Add the climber subsystem and button bindings.

### CODE REVIEW

**Review by FireShredder24 — COMMENTED** [review]
As far as I can tell, this is good.  It's obviously meant to be used with a non-instant Command, which will require some discussion, but otherwise it looks good.

File: src/main/java/frc/robot/subsystems/ClimberSubsystem.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+  public void stow() {
+    if (limit.get())
+      stop();
+    else
+      climberSP.set(CLIMB_STOW);
+  }
+
+  public void climb() {
+    if (encoder.get() > MoPrefs.getClimberEncoderLimit())
```
Comment (jordan-powers) [inline src/main/java/frc/robot/subsystems/ClimberSubsystem.java]: So we're assuming the climbing encoder will always have a valid zero before we start climbing?

**Review by jordan-powers — CHANGES_REQUESTED** [review]

**Review by FireShredder24 — COMMENTED** [review]
Looks good.

**Review by FireShredder24 — DISMISSED** [review]
Ready to merge as far as I can tell.

**Review by FireShredder24 — APPROVED** [review]
Looks good to me.

**Review by jordan-powers — APPROVED** [review]
Looks good

**Pull request merged**